### DISCUSSION TITLE
How do I get the module name of a plug-in

### DISCUSSION THREAD

**Discussion by fgheng** [wbthomason/packer.nvim#1154] [Q&A] [answered]
I want to use packer's lazy load by setting a plugin's module name, but I do not know where to get the module name

**Comment by max397574** [comment, marked as answer]
it's about lua-modules
read nanotees lua guide to know what that is
basically it's just the name of the folder inside `lua/` or the name of the file if there is just `lua/<name>.lua`

**Reply by fgheng** [reply]
thank you very much
